src/section-isolating-a-linear-variable.ptx

```diff
@@ -62,94 +64,107 @@
         </p></remark>
 
         <example xml:id="example-solve-for-variable-profit-revenue-cost">
-            <p>Solve for <m>R</m> in <m>P=R-C</m>. (This is the relationship between profit, revenue, and cost.)</p>
-
-            <p>To solve for <m>R</m>, we first want to note that <m>C</m> is <em>subtracted</em> from <m>R</m>. To <q>undo</q> this, we will need to <em>add</em> <m>C</m> to each side of the equation:
-                <md>
-                    <mrow>P\amp=\attention{R}-C</mrow>
-                    <mrow>P\addright{C}\amp=\attention{R}-C\addright{C}</mrow>
-                    <mrow>P+C\amp=\attention{R}</mrow>
-                    <mrow>R\amp=P+C</mrow>
-                </md>
-            </p>
+            <statement>
```

Comment:
I want to check in that you are not under the impression that this _needs_ to be separated by statement and solution. Often with an early example in the section, or with an example that has a short solution, we don't separate it out. I don't know where that line is drawn exactly.